bihire/property_pro_lite
# responsive navbar

#### What does this PR do?
- adds changes appearance when to shrank to a certain width
- changes the display of navigation items
- doesn't interfere in user's content

#### Description of Task to be completed
- A user should be able to adjust his navigation bar screen to every different size of the device

#### How should this be manually tested?
- clone the repo and switch to the branch ft-signup-123456
- start the server
- use the url `http://localhost:8081/home/*. ` to open the every page
- enter details in field then click enter
- you should be successfully redirected to any requested page with an adjustable navbar

#### Any background context you want to provide?
- screenshot

#### What are the relevant pivotal tracker stories?
[#167158791](http://pivotaltracker.com)

#### Screenshots (if appropriate)
![image](https://user-images.githubusercontent.com/37307172/60867175-85492400-a22a-11e9-9bb6-9893283f1f14.png)
